nss-day-cohort-60/national-parks-api
# Client can now create, update, and delete from events, event_registration, campgrounds, and campground_reservations

# Description
Added create, update, and delete functions and implemented create_resource sql helper function into events, event_registration, campgrounds, and campground_reservations 

# Steps to test
- Fetch the vs-method-mapper branch and switch to it
- Start/restart debugger
- Send a GET request to campgrounds and copy a single dictionary from the response list
- Switch to the POST function, and paste the copied dictionary into the Body field.  Remove the "id" line. 
- Send a POST request to add a campground (e.g. http://localhost:8088/campgrounds)
- Verify that a 201 status code and the new dictionary are received
- Copy the dictionary again, and switch to the PUT function. 
- Paste the dictionary into the Body field.  Change any value (except the "id").
- Send a PUT request to update the campground (e.g. http://localhost:8088/campgrounds/1)
- Verify that a 204 status code was received and switch back to the GET function.
- Request all campgrounds, and verify the dictionary was updated
- Send a DELETE request to delete the dictionary (e.g. http://localhost:8088/campgrounds/1)
- Request all campgrounds, and verify the dictionary was removed from the list

Repeat process for the following GET resources:
http://localhost:8088/events
http://localhost:8088/event_registration
http://localhost:8088/camping_reservations
